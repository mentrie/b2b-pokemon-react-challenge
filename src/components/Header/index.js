import React from 'react';
import headerImage from '../../static/images/header-image.svg';

const Header = () => (
  <header className="header">
    <p className="header__text__first">Pok√©mon</p>
    <img src={headerImage} className="header__image" alt="logo" />
    <p className="header__text__second">Rendezvous</p>
  </header>
);

export default Header;
