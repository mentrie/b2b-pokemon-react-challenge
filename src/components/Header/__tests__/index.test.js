import React from 'react';
import Header from '../index';

test('render component', () => {
  const component = shallow(<Header />);
  expect(component).toMatchSnapshot();
});

test('render a img tag', () => {
  const component = shallow(<Header />);
  expect(component.find('img').length).toBe(1);
});
