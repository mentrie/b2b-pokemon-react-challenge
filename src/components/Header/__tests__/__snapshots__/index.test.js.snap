// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render component 1`] = `
<header
  className="header"
>
  <p
    className="header__text__first"
  >
    Pok√©mon
  </p>
  <img
    alt="logo"
    className="header__image"
    src="header-image.svg"
  />
  <p
    className="header__text__second"
  >
    Rendezvous
  </p>
</header>
`;
